### ISSUE TITLE
Access denied 403 when attempting to install this package.

### ISSUE THREAD

**Issue by gsuess** [pulumi/pulumi-aws-quickstart-vpc#24]
### What happened?

Pulumi plugin install is not working, because the URL https://get.pulumi.com/releases/plugins/pulumi-resource-aws-quickstart-vpc-v0.0.2-darwin-arm64.tar.gz is not accessible. (403 access denied).

### Steps to reproduce

Create a typescript pulumi project and install the package:

```
$ npm install @pulumi/aws-quickstart-vpc
```

Run `pulumi up` and follow the instructions it gives about missing plugin `aws-quickstart-vpc`:

<img width="1675" alt="image" src="https://user-images.githubusercontent.com/1558575/185108685-87525196-1b44-462d-9a4d-9c046ee4341f.png">

```
$ pulumi plugin install resource aws-quickstart-vpc v0.0.2
```

### Expected Behavior

Installs the plugin so that the error no longer appears when running `pulumi up`.

### Actual Behavior

<img width="1675" alt="image" src="https://user-images.githubusercontent.com/1558575/185109157-789333d1-d3b7-40ca-846b-3bbb9d683a1e.png">

### Output of `pulumi about`

CLI          
Version      3.37.2
Go Version   go1.17.12
Go Compiler  gc

Plugins
NAME    VERSION
nodejs  unknown

Host     
OS       darwin
Version  12.5
Arch     arm64

This project is written in nodejs: executable='/Users/gsuess/.nvm/versions/node/v18.7.0/bin/node' version='v18.7.0'

Current Stack: Qualifyze/gsuess-dev

Found no resources associated with Qualifyze/gsuess-dev

Found no pending operations associated with Qualifyze/gsuess-dev

Backend        
Name           pulumi.com
URL            https://app.pulumi.com/gsuess
User           gsuess
Organizations  gsuess, Qualifyze

Dependencies:
NAME                                    VERSION
@pulumi/aws-quickstart-aurora-postgres  0.0.3
@typescript-eslint/eslint-plugin        4.33.0
eslint-plugin-prettier                  3.4.1
husky                                   7.0.4
@changesets/changelog-github            0.4.6
@pulumi/aws-quickstart-vpc              0.0.2
jest                                    26.6.3
shx                                     0.3.4
ts-jest                                 26.5.6
ts-node                                 10.9.1
@changesets/cli                         2.24.3
@typescript-eslint/parser               4.33.0
eslint                                  7.32.0
lint-staged                             11.2.6
eslint-config-prettier                  8.5.0
is-ci                                   3.0.1
prettier                                2.7.1
typescript                              4.7.4

Pulumi locates its logs in /var/folders/_j/n3zrsq6n373c_18tsgvfs2yc0000gn/T/ by default

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by gsuess** [comment]
I am guessing that the problem is related to the fact that it's an arm64 architecture for which you are lacking builds.

**Comment by djsd123** [comment]
Also getting this 😞 .  Thanks for raising @gsuess.  

I'm using an Macbook Pro M1. Until this gets resolved I worked around this. However, not ideal and produces a warning.

```
warning: resource plugin aws-quickstart-vpc is expected to have version >=0.0.2, but has ; the wrong version may be on your path, or this may be a bug in the plugin
```

### Use make to build for your platform

Init go mod
```
$ make
```

Build binary for your platform
```
$ make build
```
Move the binary to your plugins directory
```
$ mv ./pulumi-aws-quickstart-vpc/bin/pulumi-resource-aws-quickstart-vpc ~/.pulumi/plugins/resource-aws-quickstart-vpc-v0.0.2/
```

**Comment by AaronFriel** [comment]
Hey folks, I've triggered a fresh build of this provider and it should produce a macOS binary for Apple Silicon.

As you have noticed, this repository is not under active maintenance, so I can't assert the fitness or quality of the release, but I will monitor our build and ensure that we're able to publish a plugin for your platform.

**Comment by AaronFriel** [comment]
This should be fixed in v0.0.5.

**Comment by djsd123** [comment]
Much appreciated @AaronFriel
